lib/rubygems/commands/setup_command.rb

```diff
@@ -442,15 +442,15 @@ def rb_files_in dir
   # for installation of bundler as default gems
   def template_files
```

Comment:
Awesome! While we're at it, can you change this method to just call `template_files_in "bundler/lib"` so we're not duplicating the logic? Thanks!

Reply:
`template_files_in "bundler/lib"` is not working this method. Because installation phase needs to template files with prefix of `bundler` like `bundler/templates/newgem/...` . But uninstallation phase doesn't need it prefix.

Reply:
sorry, in that case `template_files_in "bundler/lib/bundler"` should be equivalent?

Reply:
It's good to equivalent `template_files_in` for `template_files` method.